configure.ac

```diff
@@ -294,6 +294,17 @@ esac
 AC_ARG_WITH(sandbox-shell, AS_HELP_STRING([--with-sandbox-shell=PATH],[path of a statically-linked shell to use as /bin/sh in sandboxes]),
   sandbox_shell=$withval)
 AC_SUBST(sandbox_shell)
+if ! test -z ${sandbox_shell+x}; then
+  AC_MSG_CHECKING([whether sandbox-shell has the standalone feature])
+  # busybox shell sometimes allows executing other busybox applets,
+  # even if they are not in the path, breaking our sandbox
+  if PATH= $sandbox_shell -c "busybox" 2>&1 | grep -qv "not found"; then
```

Comment:
Won't this break cross?  Does busybox include its config file in a way we could check?  Otherwise, we'd need to have a way to opt out of this check or to disable it when cross-compiling.

Reply:
Good point! I'll disable this test when cross compiling.

Reply:
Note that the sandbox-shell isn't even guaranteed to be busybox, but should be a statically linked single executable, so there's no way to really get any info from it without running it.